### PR TITLE
feat: add june 2022 LFT videos

### PR DESCRIPTION
**Pull request by Slashgear**
## Why

👋🏻 Here is a small PR adding June 2022 videos of the LFT.

## How

- add 7 new post
- No Author on those article, I don't really know if I should put author on each article. Let me know.

### CODE REVIEW

File: _posts/2022-06-03-amateur-de-pression.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: "Amateur de pression #LFT 03/06/22"
+description: ""
+tags: [plongée, partage, lft, partage]
+comments: true
+color: rgb(251,87,66)
+youtubeId: 3KbXTzypKRY
+permalink: amateur-de-pression
+---
+
+Ivresse des profondeurs, exploration, dépassement de soi, durant ce talk Hugo Riffiod nous partage sa passion pour la plongée sous-marine.
```
Comment (pmartin) [inline _posts/2022-06-03-amateur-de-pression.md:12]:
```suggestion
Ivresse des profondeurs, exploration, dépassement de soi, durant ce talk, Hugo Riffiod nous partage sa passion pour la plongée sous-marine.
```

File: _posts/2022-06-03-comment-faire-un-trailer-video-qui-dechire-avec-les-technos-web.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: "Comment faire un trailer vidéo qui déchire avec les technos web ? #LFT 03/06/22"
+description: ""
+tags: [video, react, js, remotion,ffmpeg, lft, tech]
+comments: true
+color: rgb(251,87,66)
+youtubeId: LvaHeKiwf0o
+permalink: comment-faire-un-trailer-video-qui-dechire-avec-les-technos-web
+---
+
+Un jour, alors que Mickaël Alves était fraichement arrivé à Bedrock, celui-ci a eu le malheur de demander à Antoine Caron ce sur quoi il bossait entre midi et deux et qui semblait fort l'amuser. Quelle erreur du dev Franco-Portugais ! :scream:
```
Comment (pmartin) [inline _posts/2022-06-03-comment-faire-un-trailer-video-qui-dechire-avec-les-technos-web.md:12]:
```suggestion
Un jour, alors que Mickaël Alves était fraichement arrivé à Bedrock, il a eu le malheur de demander à Antoine Caron sur quoi il bossait entre midi et deux, qui semblait fort l'amuser. Quelle erreur du dev Franco-Portugais ! :scream:
```

File: _posts/2022-06-03-comment-faire-un-trailer-video-qui-dechire-avec-les-technos-web.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Il ne se doutait pas encore de la folie de son nouveau tech lead."J'essaie de générer des vidéos en MP4 à partir de composants React, tu veux voir ?"
```
Comment (pmartin) [inline _posts/2022-06-03-comment-faire-un-trailer-video-qui-dechire-avec-les-technos-web.md:13]:
```suggestion
Il ne se doutait pas encore de la folie de son nouveau tech lead : *« J'essaie de générer des vidéos en MP4 à partir de composants React, tu veux voir ? »*
```

File: _posts/2022-06-03-connaissez-vous-cache°cache.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+title: "Connaissez vous Cache°Cache ? #LFT 03/06/22"
+description: ""
+tags: [swift, ios, lft, tech]
+comments: true
+color: rgb(251,87,66)
+youtubeId: zZnnW_h78-s
+permalink: connaissez-vous-cache°cache
+---
+
+Petite découverte d’un pattern de composition en utilisant une librairie de cache comme exemple présentée par notre expert Sebastien Drode.
```
Comment (pmartin) [inline _posts/2022-06-03-connaissez-vous-cache°cache.md:12]:
```suggestion
Petite découverte d’un pattern de composition en utilisant une librairie de cache comme exemple, présentée par notre expert Sebastien Drode.
```

File: _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+title: "Créer un jeu vidéo en moins d’une heure sur Unity #LFT 03/06/22"
+description: ""
+tags: [diy, livecoding, brewing, lft, tech]
+comments: true
+color: rgb(251,87,66)
+youtubeId: qEiUSoZbQ68
+permalink: creer-un-jeu-video-en-moins-dune-heure-sur-unity
+---
+
+Julie Nginn nous présente en livecoding la construction d'un jeux video.
```
Comment (pmartin) [inline _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md:12]:
```suggestion
Julie Nginn nous présente une introduction au moteur de jeu Unity, en *livecodant* la construction d'un jeux video.  
```

File: _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+Il s’agit d’une introduction au moteur de jeu Unity.
```
Comment (pmartin) [inline _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md:13]: Supprimer cette ligne ?

File: _posts/2022-06-03-la-pression-je-ne-la-subis-pas-je-la-fais.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+title: "La pression je ne la subis pas, je la fais #LFT 03/06/22" 
+description: ""
+tags: [homemade, diy, beer, brewing, lft, partage]
+comments: true
+color: rgb(251,87,66)
+youtubeId: BWbmj3E3G7I
+permalink: la-pression-je-ne-la-subis-pas-je-la-fais
+---
+
+Dans cette présentation Matthieu Lopez nous présente un retour d'expérience sur le brassage de bière.
```
Comment (pmartin) [inline _posts/2022-06-03-la-pression-je-ne-la-subis-pas-je-la-fais.md:12]:
```suggestion
Dans cette présentation, Matthieu Lopez nous présente un retour d'expérience sur le brassage de bière.  
```

File: _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md (hunk 6 above)
Comment (Slashgear) [reply _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md:13]:
```suggestion
```

File: _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+title: "Créer un jeu vidéo en moins d’une heure sur Unity #LFT 03/06/22"
+description: ""
+tags: [diy, livecoding, brewing, lft, tech]
+comments: true
+color: rgb(251,87,66)
+youtubeId: qEiUSoZbQ68
+permalink: creer-un-jeu-video-en-moins-dune-heure-sur-unity
+---
+
+Julie Nginn nous présente une introduction au moteur de jeu Unity, en *livecodant* la construction d'un jeux video.
```
Comment (NastasiaSaby) [inline _posts/2022-06-03-creer-un-jeu-video-en-moins-dune-heure-sur-unity.md:12]:
```suggestion
Julie Nginn nous présente une introduction au moteur de jeu Unity, en *livecodant* la construction d'un jeu video.  
```

File: _posts/2022-06-03-🍪-its-cookie-jar-time-🍪.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+title: "🍪 It’s Cookie Jar Time 🍪 #LFT 03/06/22"
+description: ""
+tags: [UX, lft, tech]
+comments: true
+color: rgb(251,87,66)
+youtubeId: -rMtDI3tvDI
+permalink: 🍪-its-cookie-jar-time-🍪
+---
+
+Découvrez “Cookie Jar”, la base de donnée de connaissance utilisateurs qui centralise et documente toute l’UX research produite chez Bedrock.
```
Comment (NastasiaSaby) [inline _posts/2022-06-03-🍪-its-cookie-jar-time-🍪.md:12]:
```suggestion
Découvrez “Cookie Jar”, la base de données de connaissances utilisateurs qui centralise et documente toute l’UX research produite chez Bedrock. 
```

File: _posts/2022-06-03-errances-a-kiilopaala.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+---
+layout: video
+title: "Errances à Kiilopää #LFT 03/06/22"
+description: ""
+tags: [voyage, lft, partage]
+comments: true
+color: rgb(251,87,66)
+youtubeId: BWbmj3E3G7I
```
Comment (vgallissot) [inline _posts/2022-06-03-errances-a-kiilopaala.md:8]: Bad youtube ID that points to other LFT: "La pression je ne la subis pas, je la fais #LFT 03/06/22"

**Review by vgallissot — APPROVED** [review]
Feels weird to me that _new_ articles are published in the past :shrug: 
otherwise, it looks good to me (once the comments have been taken into account)

**Comment by Slashgear** [comment]
@vgallissot Jekyll blog only handle a single pageable entity which is `a post`. I would have really wanted to have a `__lft` entity but this would have mean a lot of Ruby code for that. :(

File: _posts/2022-06-03-errances-a-kiilopaala.md (hunk 10 above)
Comment (Slashgear) [reply _posts/2022-06-03-errances-a-kiilopaala.md:8]: nice catch

**Review by tanguyfalconnet — APPROVED** [review]

**Review by pmartin — APPROVED** [review]

**Review by valentin-claras — APPROVED** [review]

**Review by Oliboy50 — APPROVED** [review]

**Review by arthurzinck — APPROVED** [review]

**Review by vgallissot — APPROVED** [review]

**Review by christian-vdz — APPROVED** [review]

**Review by omansour — APPROVED** [review]

**Pull request merged**